linebot/utils.py

```diff
@@ -45,3 +45,25 @@ def to_camel_case(text):
     """
     split = text.split('_')
     return split[0] + "".join(x.title() for x in split[1:])
+
+
+def safe_compare_digest(val1, val2):
+    """safe_compare_digest method.
+
+    :param str or bytes val1: string or bytes for compare
```

Comment:
OK, looks good to me.

Sorry..., How about this ?
```
:param val1: string or bytes for compare
:type val1: str | bytes
```

ref:
https://github.com/line/line-bot-sdk-python/blob/master/linebot/api.py#L79-L83

And Have you signed up ICLA?
( https://feedback.line.me/enquete/public/919-h9Yqmr1u )
If you have completed, I will merge and release 1.0.2

Thanks 😄